pkg/controllers/deprovisioning/controller.go

```diff
@@ -130,7 +129,7 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 			return reconcile.Result{}, fmt.Errorf("deprovisioning via %q, %w", d, err)
 		}
 		if success {
-			return reconcile.Result{RequeueAfter: immediately}, nil
+			return reconcile.Result{}, nil
```

Comment:
Is there a suite test we need to add in here to make sure that this isn't a regression?